worker/src/core/loaders/message-worker/index.ts

```diff
@@ -248,6 +256,111 @@ const enqueueAndSend = async (): Promise<void> => {
   }
 }
 
+// refactor to use actual raw queries
```

Comment:
you mean refactor to use ORM instead of raw queries?

Reply:
modified comment to make it clearer